test/data/openapi3/normal.yaml

```diff
@@ -113,6 +113,26 @@ paths:
                 properties:
                   integer:
                     type: integer
+      parameters:
+        - name: limit
+          in: query
+          description: maximum number of characters
+          required: false
+          schema:
+            type: integer
+            format: int32
+    options:
```

Comment:
`options` is also supported in v3, as well as `trace`, so maybe it would make sense to add support for it as well? Though not sure which method could be used then for validating the `else` case for the `case-when`.